proto/cmp/types/v1alpha/service_fact.proto

```diff
@@ -51,6 +51,9 @@ message ServiceFact {
   // actually are actually a combination of several services, like the typical
   // priority boarding, which also allows for a larger carry-on bag.
   repeated ServiceFact details = 6;
+
+  // the count of the number of services required
+  int32 count = 7;
```

Comment:
For a minimum required number the field name "count" is a bit misleading.
What about `minimum_required`?

Reply:
it is not a minimum. It is like I want 2 bags of 23 kgs. I want 1 bottle of champagne. I want 24 roses...

Reply:
extend description

Reply:
replaces count with quantity